README.md

```diff
@@ -80,18 +80,19 @@ The list of global flags is:
 ```
   --help           Show help.
   --version        Show application version.
-  -c, --config="/home/raphael/.rsc"  
+  -c, --config="/home/raphael/.rsc"
                    path to rsc config file
+  -R, --retry=0    Number of retry attempts for non-successful(due to timeouts or server errors) API responses
```

Comment:
Can we be explicit here and say `Number of retry attempts for non-successful API responses (500, 503, and timeouts only)`